### DISCUSSION TITLE
How run on a server

### DISCUSSION THREAD

**Discussion by Manuelandher** [HotCakeX/Harden-Windows-Security#120] [General]
Hello hotcakex girl, I recently discovered your project and I loved it, but I have a question, how can I run this on a 2012 server without an internet connection, I'm sorry for asking basic things but I'm still learning PowerShell.
greetings.

**Comment by HotCakeX** [comment]
Hi, thank you!
To use the Harden Windows Security module, you need to use the latest version of Windows client editions. Windows Server support might be added at a later time.

You can however use the WDACConfig module on Windows Server (also requires latest available version of the OS): https://github.com/HotCakeX/Harden-Windows-Security/wiki/WDACConfig
